### PR TITLE
rp2: Adding WS5500_EVB_PICO board port for micropython

### PR DESCRIPTION
**Pull request by omogenot**
This PR will add the needed description files to add the WS5500_EVB_PICO board to create a firmware.uf2 compatible with this board.

### CODE REVIEW

**Comment by omogenot** [comment]
This is my first PR. Bear with me. Can you help me on that one? What did I wrong?

File: .gitignore [hunk 1]
```diff
@@ -46,3 +46,17 @@ genrst/
...
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+!.vscode/*.code-snippets
+
+# Local History for Visual Studio Code
+.history/
+
+# Built Visual Studio Code Extensions
+*.vsix
```
Comment (dpgeorge) [inline .gitignore:62]: please don't include these changes

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "makefile.extensionOutputFolder": "./.vscode",
+    "cmake.configureOnOpen": false
+}
```
Comment (dpgeorge) [inline .vscode/settings.json:4]: please don't include this file

File: ports/rp2/boards/W5500_EVB_PICO/readme.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Wiznet W5500-EVB-Pico
```
Comment (dpgeorge) [inline ports/rp2/boards/W5500_EVB_PICO/readme.md:1]: if this is named `board.md` then it'll appear automatically at the download page on micropython.org

File: .gitignore (hunk 1 above)
Comment (omogenot) [reply .gitignore:62]: I was caught by the VSCode IDE that added these quite automatically. What IDE do you use (even if I already have 4 different ones on my machine :-) )
I will try to amend this.

File: .vscode/settings.json (hunk 2 above)
Comment (omogenot) [reply .vscode/settings.json:4]: ditto

File: ports/rp2/boards/W5500_EVB_PICO/readme.md (hunk 3 above)
Comment (omogenot) [reply ports/rp2/boards/W5500_EVB_PICO/readme.md:1]: Sorry I don't not get this comment. I just copied the file from the WS5100S_EVB_Pico directory and updated it.

File: .gitignore (hunk 1 above)
Comment (jimmo) [reply .gitignore]: You can use whichever IDE or editor works for you, but if you need to set custom ignore paths in`repo/ .git/info/exclude`

See the section starting with "Which file to place a pattern in depends on how the pattern is meant to be used." in https://git-scm.com/docs/gitignore 

Better yet, you might be able to configure your editor not to store state in the project dir.

**Comment by omogenot** [comment]
Can anyone help with this PR? I am stuck with the check commit message and don't know how to get rid of it.

**Comment by robert-hh** [comment]
Commit message format:

topic_or_port: And a short sentence with full stop at the end.

Example:

rp2: Add WS5500_EVB_PICO board port.

**Comment by omogenot** [comment]
I'm afraid it's too late. Github won't let me do this afterwards. It was my first PR and it's a nightmare... I receive a ton of emails from GitHub and can't do anything for them. Shall I give up and delete all my PRs?
What seems obvious for you is far from being evident for new comers....
Is there any way one of the admin can force it? Or as I mentioned shall I delete all?

**Comment by robert-hh** [comment]
You can edit the PR messagein github. Besides that, you can edit the commit messages on your PC and then run a 

git push -f <your remote> <your_branch>

to update your PR.

For you local changes, use 

git rebase -i <commit nr>

git rebase is a little bit tricky. Before using it, check the help files. When I run a `git rebase -i`, I make a copy of the repository first.

If you just want to change the last commit message, call

git commit --amend

**Comment by omogenot** [comment]
Well. I give up.

**Pull request closed without merging**